source/device/tensorrt/trt_executor.cc

```diff
@@ -34,30 +34,90 @@ EXPORT_FINISH
 
 #include <NvInferRuntime.h>
 
+#include <limits>
+#include <fstream>
+
 
 static Logger gLogger(nvinfer1::ILogger::Severity::kERROR);
 
 
+
+void SaveEngine(const std::string& fileName, nvinfer1::ICudaEngine* mEngine)
+{
+    if (fileName == "")
+    {
+        std::cout << ("empty engine file name, skip save\n");
```

Comment:
```suggestion
        TLOG_ERR("Tengine: Empty engine cache file name, skip saving.\n");
```
shoud error in function. function should suppose calling function is badly needed, and the resource should be ready.
then use TLOG_XXX functions to record info.